posts/2024-12-10_why-gno/README.md

```diff
@@ -0,0 +1,102 @@
+    - The chain should incentivize thoughtful content creation, moderation, and active participation in the community; but not block out content, so that the platform can be truly "neutral".
+      - There can and should be algorithms that show users interesting content while blocking out what they don't want to see. But users need to be able to choose between different algorithms and understand what each one is targeting; rather than solely signing up for the one that maximises "engagement" or "screen-time".
+- Why gno.land can achieve it:
+  - By solving the incentive problem, we ensure that the chain can continue to work with a specific vision while being hard to subvert. The governance aims to expand the blockchain's potential and usages in real applications, rather than the capital of its investors.
+  - By solving the technical problem, we can actually create a platform with the properties described above.
+  - gno.land pays attention to creating scalable data structures, that can host the content of the platform and enable free and universal access to it.
+
+## The Incentive Problem
+
+> Establishing Anti-Subversion Governance.
+
+Most blockchain incentive schemes focus primarily on rewarding validators for securing consensus and maintaining network integrity.  In order to build larger systems which are not just financially focused, we need to incentivize non-validators who nonetheless contribute to the advancement of the project, and make sure the chain governance is philosophically aligned, rather than just looking out for individual gains.
+
+- Exclusively rewarding a chain's validators is a good way to secure the chain; but not a good basis to create larger ecosystems outside of financial transactions.
+  - The Proof of Stake model has been successful in securing blockchains; but it has not been successful in creating incentives for using and advancing a blockchain, only for providing validation services.
+  - Additionally, as governance is most often tied to the stake of each user, control of the chain can easily be subverted by powerful and rich actors who want to acquire sizable portions of the blockchain.
+  - Similarly, Proof of Work puts the incentives in the hands of those who have the resources and the means to have cost-effective machines, incentivizing those who have the most resources or those who have good, insider access to primary resources (chips and electricity).
+- Blockchains allow us to create effective economic models for building the internet which are fair to the participants involved in making it.
+  - We should pay creators making great content.
+  - We should pay moderators who keep communities focused and true to their spirit.
+  - We should pay those contributing to collecting and creative collective silos of knowledge like wikis, and verifying its information and accuracy.
```

Comment:
Collecting what? might be a bit vague

Reply:
I believe I meant information. Because there's a repetition of the same above, I'm shortening this line